_jobs/software-engineer.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: job_post
+title: Compiler LLC - Software Engineer - W2
```

Comment:
Let's just make this say `Software Engineer - W2`, don't need Compiler in there again

Reply:
Did this as a part of title de-duplication in 20be3e54af1a0329f9fb9bd7040b887053c47540